.github/ISSUE_TEMPLATE/20-docs-issue.md

```diff
@@ -0,0 +1,46 @@
+Thank you for submitting a TFX documentation issue. Per our GitHub
+policy, we only address code/doc bugs, performance issues, feature requests, and
+build/installation issues on GitHub.
+
+The TFX docs are open source! To get involved, read the documentation
+contributor guide: https://www.tensorflow.org/community/contribute/docs
+
+## URL(s) with the issue:
+
+Please provide a link to the documentation entry, for example:
+https://www.tensorflow.org/tfx/guide
+
+## Description of issue (what needs changing):
+
+### Clear description: 
```

Comment:
This is a bit confusing to me. Do we need to include these two sentences?

Reply:
Line number 13 is the main heading and line numbers 15-44 are the sub headings. But we can remove it if it's really confusing. Please confirm.